### PR TITLE
Added Ethereum plugin installation step in Readme

### LINKED ISSUE #19: Missing ethereum plugin dependency
The installation of the needed Ethereum ledger plugin is missing in the Readme file as needed step and leads to failure of the program.

### PR DESCRIPTION
**Pull request by danielksan81**
Signed-off-by: Daniel Kunz <daniel.kunz@de.bosch.com>

resolves #19

### CODE REVIEW

File: EoT-Agents-Manifacturing-Marketplace/README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Setup of smart contracts and testnet as described [here](../service-directory/Readme.md).
 - [Fetchai](https://fetch.ai/) AEA version 1.1.1
+- Installation of [Ethereum ledger plugin](https://pypi.org/project/aea-ledger-ethereum/) in AEA using `pip install aea-ledger-ethereum` in the pipenv shell environment
```
Comment (l-wegner) [inline EoT-Agents-Manifacturing-Marketplace/README.md:7]: Actually this is installed through the [Pipfile](https://github.com/boschresearch/open-eot-agents/blob/8768450b3e86a1e012df6851276f0cb0d4f27302/EoT-Agents-Manifacturing-Marketplace/Pipfile#L8).

Comment (danielksan81) [reply EoT-Agents-Manifacturing-Marketplace/README.md:7]: Ah, you are right. Then we need to update that step in the Readme file, currently it is a mixture between all different approaches...

Comment (danielksan81) [reply EoT-Agents-Manifacturing-Marketplace/README.md:7]: I got my error, I had an additional Pipfile outside which I used...
Will update the Readme as well as the Pipfile to use aea 1.1.1

File: EoT-Agents-Manifacturing-Marketplace/README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## Prerequisites
 
 - Setup of smart contracts and testnet as described [here](../service-directory/Readme.md).
-- [Fetchai](https://fetch.ai/) AEA version 1.1.1
+- Installation of [Fetchai](https://fetch.ai/) AEA version 1.1.1 and the needed [Ethereum ledger plugin](https://pypi.org/project/aea-ledger-ethereum/). This can be done either manually or with the provided pipenv environment ([Pipfile](Pipfile)). Using [pipenv](https://github.com/pypa/pipenv) execute `pipenv install` in this folder, where the corresponding [Pipfile](Pipfile) is located.
```
Comment (l-wegner) [inline EoT-Agents-Manifacturing-Marketplace/README.md:6]: Version 1.1.1? Pipfile contains 1.1.0

**Review by l-wegner — CHANGES_REQUESTED** [review]

File: EoT-Agents-Manifacturing-Marketplace/README.md (hunk 2 above)
Comment (danielksan81) [reply EoT-Agents-Manifacturing-Marketplace/README.md:6]: omg, will fix it

**Review by l-wegner — APPROVED** [review]

**Pull request merged**